packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/v6/useEditorCommands.ts

```diff
@@ -38,13 +38,22 @@ const useEditorCommands = (props: Props) => {
 		};
 
 		return {
-			// eslint-disable-next-line @typescript-eslint/no-explicit-any -- Old code before rule was applied
-			dropItems: async (cmd: any) => {
+			dropItems: async (cmd: DropCommandValue) => {
+				let pos = cmd.pos && editorRef.current.editor.posAtCoords({ x: cmd.pos.clientX, y: cmd.pos.clientY });
```

Comment:
`posAtCoords` is how [CodeMirror internally determines the drop location for content](https://github.com/codemirror/view/blob/49e68f7cbe0440501fa666c600639fe281109d90/src/dropcursor.ts#L79) &mdash; this makes the drop location of content processed by Joplin match CodeMirror's drop cursor.